test/kitchen/tasks/run-test-kitchen.sh

```diff
@@ -173,10 +172,12 @@ for attempt in $(seq 0 "${KITCHEN_INFRASTRUCTURE_FLAKES_RETRY:-2}"); do
 
   if [ "$result" -eq 0 ]; then
       # if kitchen test succeeded, exit with 0
```

Comment:
This comment line is useless, it could be removed

Reply:
Deleted!